<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1065</width>
    <height>832</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="ImageView_camera" name="CamViewer" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1031</width>
     <height>721</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="STOPButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>770</y>
     <width>80</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>STOP</string>
   </property>
  </widget>
  <widget class="QPushButton" name="WhiteBalanceButton">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>770</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>WhiteBalance</string>
   </property>
  </widget>
  <widget class="QPushButton" name="FlipXButton">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>770</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Flip X</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AutoGainButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>770</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>AutoGain</string>
   </property>
  </widget>
  <widget class="QPushButton" name="FlipYButton">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>770</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Flip Y</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AutoExposeButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>770</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>AutoExpose</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageView_camera</class>
   <extends>QWidget</extends>
   <header location="global">ImageViewer</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
